gulp-utils.js

```diff
@@ -50,17 +56,30 @@ function getAsc() {
         if (!fs.existsSync(path)) {
             return null;
         }
-
         return fs.readFileSync(path).toString("utf8");
       },
       writeFile: (filename, contents) => {
         const name = filename.startsWith("../") ? filename.substring(3) : filename;
+        if (fs.mkdirSync){
+          mkdirp(name);
+        }
         fs.writeFileSync(name, contents);
       },
-      listFiles: () => []
+      listFiles: (dirname, baseDir) => {
+        if (fs.readdirSync == undefined){
+          return [];
```

Comment:
hmm, if it's ok to remove empty list from `listFiles` and it doesn't break anything – why do we need real implementation at all?